openMediaVideoConferencing/readme.md

```diff
@@ -0,0 +1,10 @@
+# openMediaVideoConferencing.html
+Basic demo of open media conversation routing.
+
+* All errors are written to the console.
+* Login first with username and password.
+* Post login - click on the start chat button.
+* An open media conversation is routed to an agent with targetUri = https://appear.in/${id}
+* If the conversation is created successfully, the window should navigate to https://appear.in/${id} page - this means you have created and joined a video conferencing room on appear.in
+* When the agent accepts the conversation the agent would join the same room i.e. https://appear.in/${id}. The two sides (customer and agent) could now communicate.
+
```

Comment:
We should chat about the sample name.  Is openMediaVideoConferencing the right name?  Maybe we call it just openMedia to start.  Or openMediaAppearIn?  Let's round up and chat when you've updated the other comments.